src/main/webapp/templates/postTemplate.jsp

```diff
@@ -25,13 +25,14 @@
                 <%
                     }
                 %>
-                <a href="/AttachmentServlet"><i class="fas fa-edit mr-2"></i></a>
-                <a href="/AttachmentServlet"><i class="fas fa-trash mr-2"></i></a>
+                <a href="PostEditServlet?postId=<%= post.get("postID") %>"><i class="fas fa-edit mr-2"></i></a>
+                <a href="PostServlet?action=delete&postID=<%= post.get("postID") %>"><i class="fas fa-trash mr-2"></i></a>
             </div>
         </div>
```

Comment:
It doesn't show up in the changes i just highlighted this but
can you also add the download attachment link for the attachment icon?

Reply:
Attachment download linked added :)